FirebaseStorage/Tests/SwiftIntegration/StorageIntegration.swift

```diff
@@ -379,6 +379,35 @@ class StorageIntegration: XCTestCase {
     waitForExpectations()
   }
 
+  func testSimpleGetDataWithCustomCallbackQueue() {
```

Comment:
Also add to `./FirebaseStorageSwift/Tests/Integration/StorageIntegration.swift` with result type API.

In the future, it's probably sufficient to only update the other version.

Reply:
done & noted!